app/helpers/currency-symbol.js

```diff
@@ -0,0 +1,11 @@
+import Ember from 'ember';
+import { find } from 'lodash';
+import { paymentCurrencies } from 'open-event-frontend/utils/dictionary/payment';
+
+const { Helper } = Ember;
+
+export function currencySymbol(params) {
+  return find(paymentCurrencies, ['code', String(params)]).symbol;
```

Comment:
Instead of `String(params)` ... Just do a `params[0]`. Don't unnecessarily do a type cast